doc/api/process.md

```diff
@@ -1103,6 +1103,25 @@ and [Cluster][] documentation), the `process.connected` property will return
 Once `process.connected` is `false`, it is no longer possible to send messages
 over the IPC channel using `process.send()`.
 
+## `process.constrainedMemory()`
```

Comment:
Would it make more sense for this to be exposed on `os` so that it sits alongside apis like `os.freemem()`, `os.totalmem()` or is it really more oriented towards this specific process?

Reply:
I think this API is related to process, my understanding is that each process has its own value.

Reply:
Ok